partner_multi_company/tests/test_partner_multi_company.py

```diff
@@ -133,10 +133,9 @@ def test_uninstall(self):
         uninstall_hook(self.env.cr, None)
         rule = self.env.ref("base.res_partner_rule")
         domain = (
-            "['|','|',"
-            "('company_id.child_ids','child_of',[user.company_id.id]),"
-            "('company_id','child_of',[user.company_id.id]),"
-            "('company_id','=',False)]"
+            "['|', '|', ('partner_share', '=', False),"
+            "('company_id', 'in', company_ids),"
+            "('company_id', '=', False)]"
```

Comment:
Please explain this change.

Reply:
@yajo 
As stated above, Odoo modified the record rule, so we adjusted it here to make the test succeed.